spec/ddoc.dd

```diff
@@ -1043,7 +1043,7 @@ $(P
     cross-site scripting).
 )
 
-$(H2 Links to D documentation generators)
+$(H2 $(LNAME2 links_to_d_documentation_generators, Links to D documentation generators))
 
 $(P
     A list of current D documentation generators which use Ddoc
```

Comment:
Can you give "Comparing D Immutable and Const with C++ Const" the same treatment?